packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -179,16 +178,21 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
 
   public buildRetrievePackagesSourceCommand(
     data: IsvDebugBootstrapConfig,
-    packageNames: string[]
+    packageName: string
   ): Command {
     return new SfdxCommandBuilder()
       .withDescription(
-        nls.localize('isv_debug_bootstrap_step6_retrieve_packages_source')
+        nls.localize(
+          'isv_debug_bootstrap_step5_retrieve_packages_source',
+          packageName
+        )
       )
       .withArg('project:retrieve:start')
-      .withFlag('--target-metadata-dir', this.relativeMetdataTempPath)
-      .withFlag('--package-name', packageNames.join(','))
+      .withFlag('--package-name', packageName)
       .withFlag('--target-org', data.sessionId)
+      .withFlag('--target-metadata-dir', this.relativeInstalledPackagesPath)
+      .withArg('--unzip')
+      .withFlag('--zip-file-name', packageName)
```

Comment:
ohhhhh so it's still zipped over the wire, but the CLI is unzipping for us.  Nice nice.  That makes total sense.  I was confused about how we were getting out of the unzip game.